src/App.js

```diff
@@ -1,9 +1,28 @@
 import "./App.css";
-
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/header/Header";
+import Home from "./pages/home/home";
+import ProductsPage from "./pages/products/products-combine-page";
+import WishlistPage from "./pages/wishlist/wishlsit";
+import LoginPage from "./pages/login/login";
+import ProfilePage from "./pages/profile/profile";
+import SignUpPage from "./pages/signup/signup";
+import CartPage from "./pages/cart/cart";
+import Mockman from "mockman-js";
```

Comment:
Suggestion : You can read about barrel exports and absolute paths, it will make the export paths shorter

Reply:
yeah sure, next time I'll keep that in mind